### DISCUSSION TITLE
Здоровье на карте

### DISCUSSION THREAD

**Discussion by Shootnik01** [Cosmic-Overlord/TerraGov-Marine-Corps#298] [Ideas]
Сделать ксеносам (хз, для маров возможно ли) небольшую точку, рядом с их значокм на карте, которая горит ярко-зеленым, зеленым, желтым, оранжевым, ярко-красным и бордовым для 100% 99-85% 84-50% 49-20% 20-0% и в крите соответственно. Точки в крите мигают, чтобы было заметнее

**Comment by Shootnik01** [comment]
Либо рамочкой, но тогда рамочкой жирной и понятной, что это хп, а не просто украшение

**Comment by fe-ed** [comment]
Прикольная идея, но такое скорее всего только медикам стоило бы дать

**Reply by fe-ed** [reply]
(ксеносам зайдет 99%)
